nats-streaming-server.go

```diff
@@ -195,6 +198,9 @@ func parseFlags() (*stand.Options, *natsd.Options) {
 	flag.Int64("file_fds_limit", stores.DefaultFileStoreOptions.FileDescriptorsLimit, "FileStoreOpts.FileDescriptorsLimit")
 	flag.Int("io_batch_size", stand.DefaultIOBatchSize, "IOBatchSize")
 	flag.Int64("io_sleep_time", stand.DefaultIOSleepTime, "IOSleepTime")
+	flag.String("ft_group", "", "FTGroupName")
+	flag.Int("ft_quorum", 1, "FTQuorum")
```

Comment:
In the command line output, the server's Options description and README, I document this parameter as: `Number of servers needed in order to elect a leader`. Let's see if Peter objects too and we will see if we revert to specify group size instead (again, my problem with that is that it sounds like you define a rigid group and yet, you don't know how many servers will be needed for the election unless you do the math: size/2+1).